daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -913,3 +935,70 @@ class DaffodilXMLReader private[sapi] (xmlrdr: SDaffodilXMLReader) extends org.x
    */
   def parse(arr: Array[Byte]): Unit = xmlrdr.parse(arr)
 }
+
+/**
+ * Read in an infoset from any SAX Reader for unparsing
+ */
+class DaffodilInputContentHandler private[sapi] (sContentHandler: SDaffodilInputContentHandler)
+  extends ContentHandlerProxy {
+
+  override val contentHandler: org.xml.sax.ContentHandler = sContentHandler
```

Comment:
I think this should be private/protected. I think even though the base class defines it as protected, I think not making it protected here will make it public, which we probably don't want.